Sources/Tools/MaxMain/MaxCompat.h

```diff
@@ -78,18 +92,35 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #   define SetParamBlock2Controller(pb, id, tab, ctl) pb->SetControllerByID(id, tab, ctl)
 #endif // MAX_VERSION_MAJOR
 
-#if MAX_VERSION_MAJOR <= 11 // max 2009. Just a guess, really. 2010 doesn't need this function.
-#   define INIT_CUSTOM_CONTROLS(instance) InitCustomControls(instance)
+#if MAX_VERSION_MAJOR <= 14 // Max 2012
+#   define p_end end
```

Comment:
I don't see this in the `else` case -- is `p_end` something that's defined by newer SDKs?

Reply:
Yes, Max 2013 SDK moves from `end` to `p_end` because of conflicts with stdlib